Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -1,37 +1,26 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight">
+    <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.Bridge">
```

Comment:
I like this, this was on our refactoring TODO, but this will really require us to check every screen!

@harambasicluka @ralfgehrer This PR needs to be reviewed by more than 2 people.

Reply:
@mtwalli  I added the label "author merge" so you have full control and can merge it once several ppl reviewed it.